.github/workflows/main.yml

```
@@ -0,0 +1,77 @@
+name: build
+
+on: [push, pull_request]
+
```

Also adds a schedule
```suggestion
schedule:
  - cron: 0 2 * * *
```

---

Done